lib/cc_plugin_imos/cc_plugin_imos/imos.py

```diff
@@ -177,20 +177,52 @@ def check_naming_authority(self, dataset):
 
     def check_data_centre(self, dataset):
         """
-        Check the global data centre attribute and ensure it has value
-        'eMarine Information Infrastructure (eMII)'
+        Check the global data_centre attribute is either
+          - 'eMarine Information Infrastructure (eMII)', OR
+          - 'Australian Ocean Data Network (AODN)'
         """
         ret_val = []
-        result_name = ('globalattr', 'data_centre', 'check_attributes')
-        result = check_value(('data_centre',),
-                                "eMarine Information Infrastructure (eMII)",
-                                IMOSCheck.OPERATOR_EQUAL,
-                                dataset,
-                                IMOSCheck.CHECK_GLOBAL_ATTRIBUTE,
-                                result_name,
-                                BaseCheck.HIGH)
+        result_name = ('globalattr', 'data_centre')
+        accepted_values = ('eMarine Information Infrastructure (eMII)',
+                           'Australian Ocean Data Network (AODN)')
+        reasoning = ''
+
+        data_centre = getattr(dataset, 'data_centre', '')
+        if not data_centre:
+            return [Result(BaseCheck.HIGH, False, result_name,
+                           'Missing attribute data_centre')]
+
+        passed = data_centre in accepted_values
+        if not passed:
+            reasoning = "Attribute data_centre should be either " \
+                        "'eMarine Information Infrastructure (eMII)' " \
+                        "or 'Australian Ocean Data Network (AODN)'"
+
+        ret_val.append(Result(BaseCheck.HIGH, passed, result_name, reasoning))
+        return ret_val
 
```

Comment:
Instead of testing empty or not from line 191 and then testing content (basically the same operation done twice), maybe could just test the content like:
```Python
        passed = data_centre in accepted_values
        if not passed:
            reasoning = "Attribute data_centre should be either " \
                        "'eMarine Information Infrastructure (eMII)' " \
                        "or 'Australian Ocean Data Network (AODN)'"
            if not data_centre:
                reasoning = "Missing attribute data_centre. " + reasoning
        ret_val.append(Result(BaseCheck.HIGH, passed, result_name, reasoning))
        return ret_val
```
Would be a bit more compact and informative: if attribute is missing at least we know how to document it.